pkg/util/runner_test.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
For new files from our side, let's use the Red Hat license header

Reply:
Okay